.coveragerc

```diff
@@ -4,7 +4,6 @@ source=skbeam
 omit =
     */python?.?/*
     */site-packages/nose/*
-    *test*
```

Comment:
The tests won't count against the coverage (assuming that this is combined with the .codecov.yaml), they will be assigned their own coverage value, note that this is ripped from [matplotlib](https://github.com/matplotlib/matplotlib/blob/master/ci/codecov.yml)
We also made this [change](https://github.com/NSLS-II/databroker/pull/145#issuecomment-285515858) in [databroker](https://github.com/NSLS-II/databroker/blame/master/.codecov.yml).